pom.xml

```diff
@@ -228,7 +228,7 @@
 				<version>${maven-war-plugin.version}</version>
 			</plugin>
 		</plugins>
-
+		
```

Comment:
@urvyagrawal Will you please get rid of these whitespace modifications, so that it's easier to see what changes you are proposing?